docs-ref-conceptual/includes/query-azure-cli-concepts.md

```diff
@@ -0,0 +1,655 @@
+---
+ms.topic: include
+ms.date: 28/02/2022
+author: dbradish-microsoft
+ms.author: dbradish
+manager: barbkess
+ms.custom: devx-track-azurecli
+---
+
+Azure CLI uses queries to select and modify the output of Azure CLI commands. Queries are executed client-side on the Azure CLI command's return value. Queries are executed on the JSON output before any display formatting.
```

Comment:
That sounds good, I'll change the line in the next commit. I'll also change the word output to object to avoid implying that the output has to be in JSON format.